docs/data/charts/bars/bars.md

```diff
@@ -135,6 +135,16 @@ You can use this attribute to target elements based on their series.
 
 {{"demo": "BarGradient.js"}}
 
+### Gradients
+
+By default, a gradient's units are set to `objectBoundingBox`.
+When applied to a bar, the gradient will stretch to fill the entire size of the bar, regardless of the bar's value.
+
+Alternatively, you can set `gradientUnits` to `userSpaceOnUse`, which stretches the gradient to fill the entire size of the chart.
```

Comment:
Would probably be nice to have a few words on how to achieve that and how to account for the legend/tooltip

Reply:
> how to achieve that

Sure, I'll expand on this.

> how to account for the legend/tooltip

No need to account for those since they are outside the SVG. We only need to account for the axes sizes + margins

Reply:
Done, let me know if it's clearer or if I should rewrite it. I found it a bit hard to explain succintly